.spelling

```diff
@@ -112,6 +113,7 @@ CHN
 cleanup
 CLI
 ClusterStor
+clsxxxxx
```

Comment:
This also is something we should enclose in backticks, not put in the dictionary, IMO.

```suggestion
```